pkg/pipelines/tekton_builder.go

```diff
@@ -76,17 +76,39 @@ func createTrigger(gitOpsRepo string, env *config.Environment, svc *config.Servi
 	}
 	pipelines := getPipelines(env, svc)
 	svcRepo, err := extractRepo(svc.SourceURL)
-	return eventlisteners.CreateListenerTrigger(triggerName(svc.Name), eventlisteners.StageCIDryRunFilters, svcRepo, pipelines.Integration.Binding, pipelines.Integration.Template, svc.Webhook.Secret.Name, svc.Webhook.Secret.Namespace), err
+	return eventlisteners.CreateListenerTrigger(triggerName(svc.Name), eventlisteners.StageCIDryRunFilters, svcRepo, pipelines.Integration.Bindings[0], pipelines.Integration.Template, svc.Webhook.Secret.Name, svc.Webhook.Secret.Namespace), err
 }
 
 func getPipelines(env *config.Environment, svc *config.Service) *config.Pipelines {
+	pipeline := defaultPipelines
```

Comment:
You set this up at the start, but don't change it in the code, you're returning early.